docs/embed.md

```diff
@@ -51,6 +48,12 @@ The easiest way to create a Linux binary is to use the Docker-based builder we p
     # Copy your app
     WORKDIR /go/src/app/dist/app
     COPY . .
+    # Copy environment files explicitly to prevent possible .dockerignore issues
```

Comment:
```suggestion
    # Copy .env files explicitly to prevent possible .dockerignore issues (Symfony-specific, but you may need something similar even for non-Symfony projects)
```